pkg/emulator/go.mod

```diff
@@ -0,0 +1,3 @@
+module go-emulator
```

Comment:
This folder does not need a `.mod` file, because it is inside the larger Go module from the `.mod` file at the root.